api/helpers/email.js

```diff
@@ -50,33 +64,23 @@ exports.sendConfirmEmail = async function ( projectName, email, confirmKey) {
 
     // Send the emails to the CHES (Common Hosted Email Service)
     try {
-        const getOKRes = await axios.post(_GETOK_endpoint,
-            qs.stringify({
-                'client_id': _GETOK_CLIENTID,
-                'client_secret': _GETOK_CLIENT_SECRET,
-                'grant_type': 'client_credentials',
-            }),
-            {
-                headers: {
-                    'content-type': 'application/x-www-form-urlencoded'
-                }
-            }
-        );
-        if (getOKRes && getOKRes.data && getOKRes.data.access_token) {
+        const emailToken = await getEmailToken();
+
+        if (emailToken && emailToken.data && emailToken.data.access_token) {
```

Comment:
```suggestion
        if (emailToken?.data?.access_token) {
```

Reply:
Unfortunately, the app uses Node 10, which doesn't support the optional chaining feature.